openIt/ngsild-payloads/userProfile.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "urn:ngsi-ld:UserProfile:123456789",
+  "type": "UserProfile",
+  "userInfos": {
+    "type": "JsonProperty",
+    "json": {
+      "familyName": "Smith",
+      "givenName": "John",
+      "email": "john.smith@email.com"
+    }
+  },
+  "organization": {
+    "type": "Property",
+    "value": "Agglomération Provence Verte"
+  }
```

Comment:
if groups and groups membership are managed in KC, you can't duplicate them here (actually, Stellio already manages this info)